Source/Blazorise.Bootstrap5/Components/Button.cs

```diff
@@ -22,8 +22,8 @@ protected override void BuildClasses( ClassBuilder builder )
     {
         if ( ParentCollapseHeader?.ParentCollapse != null )
         {
-            if ( ParentCollapseHeader.ParentCollapse.InsideAccordion )
-                builder.Append( "accordion-button" );
+            // if ( ParentCollapseHeader.ParentCollapse.InsideAccordion )
```

Comment:
I think this

```
if ( ParentCollapseHeader.ParentCollapse.InsideAccordion )
    builder.Append( "accordion-button" );
```

should go away for both collapse and accordion.

Reply:
I don't understand fully the connection between collapse (collapseHeader, etc) and button. 

But this seems the full removal of "collapse stuff" from button should be done? Because it seems there is no other function for that..?

I am really not sure, please check the newest commit.

Reply:
The Button was used as a toggler for collapse. Since we have introduced AccordionToggle, the Button is not needed anymore.